articles/service-fabric/service-fabric-understand-and-troubleshoot-with-system-health-reports.md

```diff
@@ -53,6 +53,16 @@ The report specifies the global-lease timeout as the time-to-live (TTL). The rep
 * **Property**: Starts with **Neighborhood** and includes node information.
 * **Next steps**: Investigate why the neighborhood is lost, for example, check the communication between cluster nodes.
 
+### Rebuild
+**System.FM** and **System.FMM** reports an error when it detects that rebuild is stalled or stuck. Rebuild could be stuck in one of two phases:
+
+* Waiting for broadcast. FM/FMM waits for broadcast message reply from the other nodes. Investigating the network connection between the nodes in the cluster might provide more information.
```

Comment:
I would phrase in terms of "Next step: investigate whether this is network connection issue between nodes". Can you add more info here or is it out of scope? Is it because the network may be slow? nodes down? ring partitioning? smth else?